Tasks/DotNetCoreCLIV2/dotnetcore.ts

```diff
@@ -105,21 +108,43 @@ export class dotNetExe {
             } else {
                 dotnet.arg(projectFile);
             }
+            if(this.isBuildCommand()) {
```

Comment:
Can this logger be used in other commands such as: publish/pack. As these commands inherently runs dotnet build

Reply:
Yes it should be, then we will get warnings highlighted for those commands as well.